posorneg.c

```diff
@@ -0,0 +1,24 @@
+/*
+*Emma Ladouceur
```

Comment:
Nice job! Next time maybe think about handling the 0 differently - don't worry about it now because it wasn't required.